### PR TITLE
Implement f{32,64}.copysign instructions

### PR DESCRIPTION
**Pull request by chfast**
image

### CODE REVIEW

File: lib/fizzy/execute.cpp [hunk 1]
```diff
@@ -1470,6 +1470,16 @@ ExecutionResult execute(Instance& instance, FuncIdx func_idx, span<const Value>
             binary_op(stack, std::divides<float>{});
             break;
         }
+        case Instr::f32_copysign:
+        {
+            // TODO: This is not optimal implementation. The std::copysign() is inlined, but
```
Comment (axic) [inline lib/fizzy/execute.cpp:1475]: Why not implement it then? Is that planned after merging this? Did you only see this for `copysign`?

Comment (chfast) [reply lib/fizzy/execute.cpp:1475]: Only for `copysign` and `abs`. Also theoretical UB will be involved, so I prefer to have all spec tests passed before doing that.

File: test/unittests/execute_floating_point_test.cpp [hunk 2]
```diff
@@ -408,6 +408,35 @@ TYPED_TEST(execute_floating_point_types, div)
...
+    using FP = FP<TypeParam>;
+    using Limits = typename FP::Limits;
+
+    auto instance = instantiate(parse(this->get_binop_code(Instr::f32_copysign)));
+    const auto exec = [&](auto arg1, auto arg2) { return execute(*instance, 0, {arg1, arg2}); };
+
+    std::vector p_values(
+        std::begin(this->positive_special_values), std::end(this->positive_special_values));
+    for (const auto x :
+        {TypeParam{0}, Limits::infinity(), FP::nan(FP::canon), FP::nan(FP::canon + 1), FP::nan(1)})
```
Comment (gumb0) [inline test/unittests/execute_floating_point_test.cpp:422]:
```suggestion
        {TypeParam{0}, Limits::infinity(), FP::nan(FP::canon), -FP::nan(FP::canon), FP::nan(FP::canon + 1), -FP::nan(FP::canon + 1), FP::nan(1), -FP::nan(1)})
```

File: lib/fizzy/execute.cpp [hunk 3]
```diff
@@ -1538,6 +1538,16 @@ ExecutionResult execute(Instance& instance, FuncIdx func_idx, span<const Value>
             binary_op(stack, fdiv<float>);
             break;
         }
+        case Instr::f32_copysign:
+        {
+            // TODO: This is not optimal implementation. The std::copysign() is inlined, but
+            //       it afects the compiler to still use SSE vectors (probably due to C++ABI)
```
Comment (axic) [inline lib/fizzy/execute.cpp:1544]: Nit: affects

Comment (chfast) [reply lib/fizzy/execute.cpp:1551]: Where is spellcheck???

Comment (axic) [reply lib/fizzy/execute.cpp]: I'm here :wave:

**Pull request merged**